contents/blog/recommended-reading.md

```diff
@@ -0,0 +1,148 @@
+
+
+![high output management](../images/tracks/high-output.jpg)
+## [High Output Management by Andrew Grove](https://www.goodreads.com/book/show/324750.High_Output_Management)
+**Recommended by [Grace McKenzie, Finance & Operations Lead](/community/profiles/718)**
+
+Andrew Grove was Intel’s third employee, third CEO, and the man who led the company to the world’s largest producer of semiconductors. He also somehow found time to write this book about organizational theory which is beloved by our chief organizer, Grace.
+
+“High Output Management is filled with light-touch and thoughtful management advice which can lead to increases in productivity,” says Grace. “In particular it advocates for the importance of gathering relevant information, making decisive decisions, and nudging teams towards a goal rather than setting a specific direction.”
+
+> **Putting it into practice:** We’ve templated [our approach to goal-setting](/handbook/company/goal-setting) so that we can make fast decisions, and not waste time discussing formats. 
+
+
+![no rules rules](../images/tracks/no-rules-rules.jpg)
+## [No Rules Rules by Reed Hastings](https://www.goodreads.com/book/show/49099937-no-rules-rules) 
+**Recommended by [James Hawkins, CEO & Co-founder](/community/profiles/71)**
+
+When it came to envisioning the sort of culture he wanted to build at PostHog, no other book was as influential for James as this exploration of Netflix’s company culture. PostHog doesn’t copy Netflix’s approach completely, but we definitely share some of the underlying philosophies. 
+
+“No Rules Rules clarified how we should totally trust each person here,” says James. “It also helped us realize just how much a smaller, stronger, and better paid group of people can accomplish…It’s just so counter to the way that other companies operate, where they focus on the whole function.”
+
+> **Putting it into practice:** We validated a lot of Netflix’s rules when [hiring our first five employees](/blog/posthog-first-five) — and we’ve rolled them into [our overall culture](/about).
+
+
+![getting real](../images/tracks/getting-real.png)
+## [Getting Real by Basecamp](https://basecamp.com/gettingreal)
+**Recommended by [David Newell, Software Engineer](/community/profiles/860)**
+
+Basecamp’s online manifesto about how they build products echoes a lot of the same principles we believe in, from the importance of iterative development to the focus on user feedback. David, who built several successful products before joining PostHog, thinks it’s a resource everyone can benefit from — not just engineers. 
+
+“It very simply talks about avoiding ‘work work’,” says David. “The focus on getting your product in front of customers as early as possible so that you avoid a bunch of implicit assumptions and sunk costs is particularly resonant.” 
```

Comment:
```suggestion
David's review: “It very simply talks about avoiding ‘work work’. The focus on getting your product in front of customers as early as possible so that you avoid a bunch of implicit assumptions and sunk costs is particularly resonant.” 
```